### PR TITLE
Design

### PR DESCRIPTION
**Pull request by Mhamad-Raad**
# Design
## In this branch I have done the followings:
- Designed the web app.
- Fixed categories bug.
### Jobs all done 🎉

### CODE REVIEW

File: src/components/Book.jsx [hunk 1]
```diff
@@ -1,27 +1,74 @@
...
+        <p className={css.category}>{ category }</p>
+        <h3 className={css.title}>{title}</h3>
+        <p className={css.author}>{author}</p>
+        <div className={css.buttons}>
+          <button type="button" className={css.edits}>Comments</button>
+          <div className={css.vLine} />
+          <button type="button" onClick={handleRemoveBook} className={css.edits}>Remove</button>
+          <div className={css.vLine} />
+          <button type="button" className={css.edits}>Edits</button>
+        </div>
```
Comment (codepantha) [inline src/components/Book.jsx:42]: - [ ] Please let's follow the design guideline and ensure that the left and right margins for each `Book` is `6.25rem`.
### Zeplin Design Guideline
image
### Your Implementation
image

File: src/components/Header.jsx [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
+              </NavLink>
+            </li>
+            <li>
+              {' '}
+              <NavLink to="Categories" className={`${css.links} ${css.disabled}`}>
+                Categories
+              </NavLink>
+            </li>
+          </ul>
+        </div>
```
Comment (codepantha) [inline src/components/Header.jsx:28]: - [ ] Please ensure that the left and right margin on the `Header` component corresponds with the Zeplin design as shown below 👇 
### Zeplin Design
image

### Your implementation
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @Mhamad-Raad 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ No linter errors
✔️ Used GitFlow

## Required Changes ♻️
_Check the comments under the review._

- [ ] I noticed that the link to your LICENSE in the README.md is broken. Kindly fix it

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @Mhamad-Raad, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**